lib/pbio/src/light/animation.c

```diff
@@ -50,10 +50,10 @@ void pbio_light_animation_start(pbio_light_animation_t *animation) {
     pbio_light_animation_list_head = animation;
 
     process_start(&pbio_light_animation_process);
-    // HACK: init timer since we don't call etimer_set()
-    timer_set(&animation->timer.timer, 0);
-    // fake a timer event to load the first cell and start the timer
-    process_post_synch(&pbio_light_animation_process, PROCESS_EVENT_TIMER, &animation->timer);
+    // Fake a timer event to load the first cell.
+    PROCESS_CONTEXT_BEGIN(&pbio_light_animation_process);
+    etimer_set(&animation->timer, 0);
+    PROCESS_CONTEXT_END(&pbio_light_animation_process);
```

Comment:
I thought I'd clean this up while touching this code, but it looks like the tests expect this to be synchronous using the aforementioned hack. We can revert this bit or add

```
    while (pbio_do_one_event()) {
    }
```
to the tests. I'll update later.